src/Modals/ModalFooter.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement, ReactNode } from 'react';
+import styled, { withTheme } from 'styled-components';
+
+interface ModalFooterProps {
+  /**
+   * the content of the Modal Footer
+   */
+  children: ReactNode;
+}
+
+
+const styledModalFooter = styled.div<ModalFooterProps>`
+  display: flex;
+  flex-direction: reverse-row;
```

Comment:
Do you mean `row-reverse` here? I'm not seeing a `reverse-row` option:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction
Also, could you clarify why we are ordering in reverse?

Reply:
From the example, it seems like we are ordering in `row` order

Reply:
Totally right, that should be `row-reverse` and I just did it wrong.

The reason for ordering in reverse is to follow the style of [Vittorio's mockups](https://github.com/seas-computing/course-planner/issues/169#issuecomment-601413769), where:
- **With only one button** the button is located in the bottom right corner
- **With two buttons** they're in the bottom right and bottom left corners (i.e., not right next to each other).
- I don't think we've specified what to do with 2+ buttons, but as of right now they'd be distributed evenly across the row.

The "reverse" part of it is less than ideal. We could use `justify-content: end;` with `flex-direction: row;`, but that would pack the items to the right, so the "cancel" button would be right next to the "confirm button" (not good). I'm open to other ideas though. Any thoughts?

Reply:
Oh I see what you're saying with the `reverse` part of this now. I think this is the best way to go